Makefile

```diff
@@ -0,0 +1,39 @@
+.PHONY: all build test run bash tag push help
+
+NAME=echoapi
+NAMESPACE=quay.io/3scale
+VERSION=centos7-to-ocp
+LOCAL_IMAGE := $(NAME):$(VERSION)
+REMOTE_IMAGE := $(NAMESPACE)/$(LOCAL_IMAGE)
+
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+THISDIR_PATH := $(patsubst %/,%,$(abspath $(dir $(MKFILE_PATH))))
+PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
+
+
+all: build
+
+build: ## Build docker image with name LOCAL_IMAGE (NAME:VERSION).
+	docker build -f $(THISDIR_PATH)/Dockerfile -t $(LOCAL_IMAGE) $(PROJECT_PATH)
+
+test: ## Test built LOCAL_IMAGE (NAME:VERSION). 
+	docker run -t --env RACK_ENV=$(ENVIRONMENT) $(LOCAL_IMAGE) rackup -D
+	docker run -t $(LOCAL_IMAGE) 3scale_backend --version
```

Comment:
Yes it is a wrong copy and paste, I actually make it right on 081f95b. Do you want I rebase the commits to get it out?